packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/NextButton/NextButton.spec.js

```diff
@@ -57,12 +57,12 @@ describe('NextButton', function () {
     })
 
     it('should not be auto-focused if props.autoFocus is false', function () {
-      expect(wrapper.find('WithTheme(Button)').props().autoFocus).to.be.false()
+      expect(wrapper.props().autoFocus).to.be.false()
```

Comment:
Same comments here about testing the values of props. I'm not really sure what that's testing.